latest/docs-ref-autogen/ad/user.yml

```diff
@@ -78,7 +78,7 @@ directCommands:
                                  [--security-enabled-only {false, true}]
   examples:
   - summary: |-
-      Get groups of which the user is a member
+      Get groups of which the user is a member.
```

Comment:
Period is automatically attached when rendering the help message: 
![image](https://github.com/user-attachments/assets/ff632b74-59e1-402f-9120-2c925251fe00)